### PR TITLE
AWS S3 blob storage

### PR DESCRIPTION
**Pull request by esuomi**
- replaces `docker run...` with `docker-compose.yml`
 - adds AWS S3 implementation for BlobStoreRepository
 - adds [LocalStack](https://www.localstack.cloud/) AWS simulator for running AWS components locally

There are some potentially breaking or at least confusing-to-user changes to how profiles are handled, plus the documentation is slightly confusing now since both the Docker Compose has optional profiles and the service itself uses profiles in a way where specific combinations of profiles are intended to work together.

As further thought, having the Google PubSub Emulator available as Docker Compose profile similar to LocalStack might also be a rather clean way to do that as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,14 +140,16 @@ Provider-specific GraphQL endpoint (replace {providerCode} with provider's codes
 
     /services/flexible-lines/{providerCode}/graphql
 
-## Netex Export
+## NeTEx Export
```
Comment (testower) [inline README.md:143]: 😍

File: init-localstack.sh
Comment (testower) [inline init-localstack.sh:1]: Maybe move this into a scripts folder

**Comment by testower** [comment]
> As further thought, having the Google PubSub Emulator available as Docker Compose profile similar to LocalStack might also be a rather clean way to do that as well.

I think it's a very interesting option!

File: src/main/java/no/entur/uttu/export/blob/LocalDiskBlobStoreRepositoryConfig.java [hunk 2]
```diff
@@ -19,15 +19,20 @@
...
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
 
 @Configuration
-@Profile({ "local", "test", "local-disk-blobstore" })
+@Profile({ "local-disk-blobstore" })
 public class LocalDiskBlobStoreRepositoryConfig {
 
   @Bean
+  @ConditionalOnMissingBean(
```
Comment (vpaturet) [inline src/main/java/no/entur/uttu/export/blob/LocalDiskBlobStoreRepositoryConfig.java:32]: what is the purpose of the conditional bean configuration here?

**Review by vpaturet — APPROVED** [review]

**Pull request merged**